persistence-api/src/main/java/io/stargate/db/datastore/schema/Keyspace.java

```diff
@@ -32,7 +32,7 @@
 
     /**
      * This list is verified in a test against SystemInfo. We don't want to use this directly though for
-     * ExternalDataStore as it shouldn't have dependencies on DSE internal classes
+     * ExternalDataStore as it shouldn't have dependencies on Cassandra's internal classes
```

Comment:
What about the header on this one?

Reply:
Talked to Brittni from Legal, she said to leave it for now while she circles w/Jason. Will wait to hear back. Did we modify any of the DSE code?

Reply:
For the most part, no.